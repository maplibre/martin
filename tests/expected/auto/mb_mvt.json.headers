content-encoding: br
content-type: application/json
etag: W/"1a3-AgILNjTcvIE3t7rN3x0S0Q=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
