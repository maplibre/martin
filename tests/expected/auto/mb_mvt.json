{
  "bounds": [
    -123.12359,
    -37.818085,
    174.763027,
    59.352706
  ],
  "center": [
    -75.9375,
    38.788894,
    6
  ],
  "description": "Major cities from Natural Earth data",
  "format": "pbf",
  "maxzoom": 6,
  "minzoom": 0,
  "name": "Major cities from Natural Earth data",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/world_cities/{z}/{x}/{y}"
  ],
  "vector_layers": [
    {
      "description": "",
      "fields": {
        "name": "String"
      },
      "id": "cities",
      "maxzoom": 6,
      "minzoom": 0
    }
  ],
  "version": "2"
}
